styles.css

```diff
@@ -0,0 +1,235 @@
+nav {
+    display: block;
+    height: 80px;
+    margin-bottom: 20px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+#logo-youtube {
+    height: 180px;
+    width: 240px;
+    margin-right: 25px;
+}
+
+#logging-img {
+    height: 50px;
+    width: 50px;
+    margin-right: 50px;
+}
+
+#searchVideos {
+    /* width: 700px;
+    height: 30px; */
```

Comment:
Same here, if you don't need this code you can remove it.

Reply:
Done it, thanks.